.gitattributes

```diff
@@ -1,4 +1,4 @@
-*   text=auto
-*.py    text eol=lf
-*.sh    text eol=lf
-*   text working-tree-encoding=UTF-8
\ No newline at end of file
+*   -text
+# *.py    text eol=lf
+# *.sh    text eol=lf
+# *   text working-tree-encoding=UTF-8
```

Comment:
Previously, we forced endline conversion to simplify development on Windows. However, this affects image files, as they get corrupted because of CRLF to LF line conversion. However, I did not delete the existing config options, because whether to disable them or not is subject to discussion.